docs/source/how_to/how_to_criterion_function.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>you need to <s>declare</s> mark your function with </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/optimagic/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>